framework/src/Volo.Abp.BlazoriseUI/BreadcrumbItem.cs

```diff
@@ -1,14 +1,16 @@
-﻿namespace Volo.Abp.AspNetCore.Components.WebAssembly
+﻿using Blazorise;
+
+namespace Volo.Abp.BlazoriseUI
 {
     public class BreadcrumbItem
     {
         public string Text { get; set; }
 
-        public string Icon { get; set; }
+        public IconName? Icon { get; set; }
```

Comment:
It would be better to have as an `object`  in case someone wants to set a name explicitly, like "fa-home". Blazorise supports this scenario and will apply the right name be it from enum or a string name.

Reply:
Done.